_sass/_va.scss

```diff
@@ -991,49 +989,44 @@ span.next {
   display: inline-block;
 }
 
-
-#experience .splash {
-  margin: 0;
-  padding: 0; 
-  
-  @media #{$small} {
-    margin: .5em 0 0 0;
-    padding: 1em 0 0 0;
-  }
+// TODO: Remove !important from this block
+// Once the specificity of #content .splash h2 block/
+// gets reduced.
+.va-headingflag {
+  margin: .5em 0 !important; 
+  padding: 0 !important;
+  margin: 0 0 2em 0 !important; 
+  background: none !important; 
+  color: $color-primary-darkest !important;
   
-  h2 {
-    &.tagline {
-      padding: 0;
-    }
+  @media #{$medium} {
+    position: absolute !important; 
+    margin: 1.35em 0 0 0 !important; 
+    z-index: 100 !important;
+  }  
     
-    padding: 0; 
-    margin: 0 0 2em 0; 
-    background: none; 
-    color: $color-primary-darkest;
-    
-    a {
-      text-decoration: none; 
-      margin: 0; 
-      color: $color-primary-darkest; 
-      background: $color-gold; 
-      border-bottom: none; 
-      padding: .5em .25em; 
+  a {
+    margin: 0 !important;
+    background: $color-gold !important; 
+    border-bottom: none !important;
+    color: $color-primary-darkest !important;
+    font-size: .9em !important;
+    line-height: 1.4em !important;
+    padding: .5em .25em; 
+    text-decoration: none; 
```

Comment:
I think this needs to be in the h2 in order to be applied?
![screen shot 2016-04-05 at 2 18 25 pm](https://cloud.githubusercontent.com/assets/1307774/14298343/5f3ca64c-fb39-11e5-895a-fdd3ab3b2ba2.png)